### PR TITLE
Feat(eos_designs): add support for 7300X3 in default platforms

### LINKED ISSUE #1860: Feat(eos_designs): support for 7300X platform
### Enhancement summary

Please add Platform 7300X3 to the EOS_Designs - default platform_settings: 

We are using these devices as Spines therefore require the management_interface: Management0 defined however don't require MLAG reload timers.

### Which component of AVD is impacted

eos_designs

### Use case example

Using 7308X3 as Spine in AVD

### Describe the solution you would like

```
platform_settings:
  - platforms: [ '7300X3' ]
    management_interface: Management0
```

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by gmuloc**
* Move the default platform_settings variable to its own file for more
  clarity

## Change Summary

add support for 7300X3 in default platforms

## Related Issue(s)

Fixes #1860

## Component(s) name

`arista.avd.<role-name>`

## Proposed changes

1. moved the default platforms settings to a separate file
2. Add the following platform (no other list matching)
    ```yaml
    platform_settings:
      - platforms: ['7300X3']
        management_interface: Management0
    ```

## How to test

reused one platform from `eos_designs_unit_test` molecule scenarios to test the settings

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by jonxstill — APPROVED** [review]
This looks good to me.  Given that we're going to be making more use of this (see https://github.com/aristanetworks/ansible-avd/pull/1758 for my current piece of work, which I'm heavily refactoring now) and may have a proliferation of platforms, having a separate file is good.  Also end users could then potentially use the combine filter to merge the defaults with their own platform-specific settings.

File: ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-platform-settings.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    reload_delay:
+      mlag: 900
+      non_mlag: 1020
+  - platforms: ['7368X4']
+    management_interface: Management0
+    reload_delay:
+      mlag: 300
+      non_mlag: 330
+  - platforms: ['7300X3']
+    management_interface: Management0
```
Comment (tgodaA) [inline ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-platform-settings.yml:40]: Would be good to add reload_delay.

Comment (tgodaA) [reply ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-platform-settings.yml:40]: Thank you for finding the values.

**Review by xaviramon — APPROVED** [review]
LGTM

**Pull request merged**